actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -38,17 +46,48 @@ def take_failed_screenshot
         end
 
         private
+          mattr_accessor :screenshot_counter
```

Comment:
This can't be a `mattr_accessor` because those tests could run in parallel and one test would influence the counter of the other.

Reply:
Good point.  If I use an instance variable here, is there a naming standard to reduce the possibility of collisions with user variables?

Reply:
Using `_` in the front.

Reply:
Updated to now use an instance variable for the counter.